go/cmd/pathwar/main.go

```diff
@@ -495,7 +495,7 @@ func main() {
 					return errcode.ErrInitDockerClient.Wrap(err)
 				}
 
-				services, err := pwcompose.Up(ctx, string(preparedCompose), composeUpInstanceKey, composeUpForceRecreate, "", nil, cli, logger)
+				services, err := pwcompose.Up(ctx, string(preparedCompose), composeUpInstanceKey, composeUpForceRecreate, "", nil, globalDebug, cli, logger)
```

Comment:
Do not pass a new variable we already have a configured logger 

You can check in the code there are examples of if conditions that use fmt.Println if logger level is debug